src/TLuaInterpreter.cpp

```diff
@@ -2422,6 +2422,16 @@ int TLuaInterpreter::closeMudlet(lua_State* L)
     return 0;
 }
 
+int TLuaInterpreter::loadWindowLayout(lua_State *L) {
+    mudlet::self()->loadWindowLayout();
```

Comment:
I think this and the next command have to return the boolean that the `mudlet` class methods return.